extension/content.js

```diff
@@ -26,15 +26,15 @@ function toggleFiles() {
 	let i = 0;
 
 	for (const el of rows) {
-		if (el.querySelector('.content a')) {
-			const fileName = el.querySelector('td.content a').innerText;
+		if (el.querySelector('.content > * > *')) {
```

Comment:
Oh, I wasn't aware of `:-webkit-any()`. I pushed an update: https://github.com/sindresorhus/hide-files-on-github/pull/44/commits/e7dd0f2d2ed52cbf288e246a9c51b1de30772376